lib/ReactViews/SidePanel/SidePanel.jsx

```diff
@@ -78,46 +81,83 @@ const SidePanel = createReactClass({
         return (
             <div className={Styles.workBench}>
                 <div className={Styles.header}>
-                    <SearchBox onSearchTextChanged={this.changeSearchText}
-                               onDoSearch={this.search}
-                               onFocus={this.startLocationSearch}
-                               searchText={searchState.locationSearchText}
-                               placeholder="Search for locations" />
+                    <FullScreenButton
+                        terria={this.props.terria}
+                        viewState={this.props.viewState}
+                        minified={true}
+                        animationDuration={250}
+                    />
+                    <SearchBox
+                        onSearchTextChanged={this.changeSearchText}
+                        onDoSearch={this.search}
+                        onFocus={this.startLocationSearch}
+                        searchText={searchState.locationSearchText}
+                        placeholder="Search for locations"
+                    />
                     <div className={Styles.addData}>
-                        <button type='button' onClick={this.onAddDataClicked} className={Styles.button}>
-                            <Icon glyph={Icon.GLYPHS.add}/>Add data
+                        <button
+                            type="button"
+                            onClick={this.onAddDataClicked}
+                            className={Styles.button}
+                        >
+                            <Icon glyph={Icon.GLYPHS.add} />Add data
                         </button>
                     </div>
                 </div>
                 <div className={Styles.body}>
                     <Choose>
-                        <When condition={searchState.locationSearchText.length > 0 && searchState.showLocationSearchResults}>
+                        <When
+                            condition={
```

Comment:
Lots of unnecessary formatting changes here making it hard to see what's actually changed.